🚀 Add Missing Functions, Convert to Kotlin, Upgrade Gradle & Target SDK 34

ssskhan opened the pull request:
Added missing functions requested by the community. Fixes #5  and #4 

Converted the entire codebase from Java to Kotlin for better readability and maintainability.

Upgraded Gradle to 8.5.2.

Updated minSdk from 16 → 21 and target/compileSdk from 29 → 34 to align with modern Android development.

ssskhan committed:
Rename .java to .kt

ssskhan committed:
version 2.0
converted to kotlin
upgraded gradle to 8.5.2
added new functions

ssskhan committed:
Update README.md

added new jitpack link as previous repo doesnt seem to be maintained anymore

ssskhan commented:
Note: It seems this repository is no longer actively maintained.
If you're looking for an updated and working version of the library, please check out my fork here:

👉 https://github.com/ssskhan/Neomorphic-FrameLayout-Android
📦 JitPack dependency:

```gradle
implementation 'com.github.ssskhan:Neomorphic-FrameLayout-Android:2.0.0'
```

Hope it helps anyone still using this awesome library! 😊

4inodev commented:
@ssskhan thanks for the contribution :) Didn't have enough time to fully look through, but looks good 👍

ssskhan commented:
Hi, I thought repo is dead so I updated the dependency with my username and added a note and unlinked the fork, kindly update the ReadME file  after making the 2.0.0 release and remove the note too above the dependency

4inodev commented:
> Hi, I thought repo is dead so I updated the dependency with my username and added a note and unlinked the fork, kindly update the ReadME file after making the 2.0.0 release and remove the note too above the dependency

Why tf would you update the user name in a PR to my repo?

4inodev commented:
@ssskhan damn dude wtf

ssskhan commented:
I thought the repo is dead so today I changed the readme file and unlinked the repo from yours. I don't know how but the very moment I was unlinking the repo you merged the request and it seems you got my last commit too where I updated the readme I am very sorry
